portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
consider creating a dedicated css file for the projects page

Reply:
If you make a dedicated CSS file, you may want to have one that's specific to this page, and another for common styles that you use throughout the site. Similar for the JS files if there will be some code reused on different pages.